Makefile

```diff
@@ -38,6 +71,40 @@ release:
 	cat ecs-proxies-deploy.yml | sed -e 's/{{ SPINE_ENV }}/ref/g' -e 's/{{ SANDBOX_MODE_ENABLED }}/0/g' > dist/ecs-deploy-ref.yml
 	cp ecs-proxies-deploy-prod.yml dist/ecs-deploy-prod.yml
 
+release-epsat:
+	mkdir -p dist
+	cp ecs-proxies-deploy.yml dist/ecs-deploy-all.yml
+	for env in internal-dev prod; do \
+		cp ecs-proxies-deploy.yml dist/ecs-deploy-$$env.yml; \
+	done
+	cp ecs-proxies-deploy-internal-dev-sandbox.yml dist/ecs-deploy-internal-dev-sandbox.yml
+	cp ecs-proxies-deploy-internal-qa.yml dist/ecs-deploy-internal-qa.yml
+	cp ecs-proxies-deploy-int.yml dist/ecs-deploy-int.yml
+	cp ecs-proxies-deploy-sandbox.yml dist/ecs-deploy-sandbox.yml
+	cp packages/tool/specification/eps-api-tool.json dist/
+	cp -Rv packages/tool/proxies dist
+	rsync -av --progress packages/tool/e2e-tests dist --exclude e2e-tests/node_modules
+
+prepare-for-api-release:
+	cp api.ecs-proxies-containers.yml ecs-proxies-containers.yml
+	cp api.ecs-proxies-deploy-prod.yml ecs-proxies-deploy-prod.yml
+	cp api.ecs-proxies-deploy.yml ecs-proxies-deploy.yml
+	cp api.manifest_template.yml manifest_template.yml
+	touch api.release
+
+prepare-for-epsat-release:
+	cp epsat.ecs-proxies-containers.yml ecs-proxies-containers.yml
+	cp epsat.ecs-proxies-deploy-int.yml ecs-proxies-deploy-int.yml
+	cp epsat.ecs-proxies-deploy-internal-dev-sandbox.yml ecs-proxies-deploy-internal-dev-sandbox.yml
+	cp epsat.ecs-proxies-deploy-internal-qa.yml ecs-proxies-deploy-internal-qa.yml
+	cp epsat.ecs-proxies-deploy-sandbox.yml ecs-proxies-deploy-sandbox.yml
```

Comment:
move files to coordinator and epsat folder and copy from there to root directory